### DISCUSSION TITLE
Disk keeps filling up

### DISCUSSION THREAD

**Discussion by HaroldPoling** [Security-Onion-Solutions/securityonion#6350] [General]
I have been loading and evaluating Security Onion 2.3.5 5 6 7 now 9 and each one except 9 so far has filled up my virtual disk I am using 300 or 400 gig disks and build from ISO.  Any Idea's?

**Comment by dougburks** [comment]
When you say `filled up`, is it staying at 90% or is it actually going to 100%?
